test/extended/util/test.go

```diff
@@ -131,12 +131,21 @@ func ExecuteTest(t ginkgo.GinkgoTestingT, suite string) {
 }
 
 func AnnotateTestSuite() {
-	testRenamer := newGinkgoTestRenamerFromGlobals(e2e.TestContext.Provider)
+	testRenamer := newGinkgoTestRenamerFromGlobals(e2e.TestContext.Provider, getNetworkSkips())
 
 	ginkgo.WalkTests(testRenamer.maybeRenameTest)
 }
 
-func newGinkgoTestRenamerFromGlobals(provider string) *ginkgoTestRenamer {
+func getNetworkSkips() []string {
```

Comment:
This really doesn't belong here, since it's making network calls deep in the guts of the initialization path.  If you want to provide plugin specific skips you need to configure that in the top level of openshift-tests (in the same place the provider is initialized) and then follow the same pattern as the cloud skips.

If you're too busy to rework this it'll be in my next pass, but I'd like this fixed soon.